docs/pages/guides/sharing-before-release.md

```diff
@@ -0,0 +1,43 @@
+---
+title: Sharing before release
+---
+
+import ImageSpotlight from '~/components/plugins/ImageSpotlight'
+
+Ultimately, you will want to publish your application to the App Store and Play Store to access the opportunities for wider distribution and to provide a familiar installation process for your users.  While you are developing though, you'll want to let other members of your team and others with a vested interet in the project see the work in progress to gather feedback and make sure you're building a quality product.
+
+## Expo Go
+
+If you are using the Expo Managed workflow, you can share your application with your team through the Expo Go app.
+
+1. Publish the latest version of your project to Expo's servers by running `expo publish`.
+2. [Invite your teammate](https://expo.io/[account]/[project]/settings/members) to the Expo account that owns the project.
+3. Have them [download the Expo Go app](https://expo.io/expo-go) onto their device.
+4. They'll be able to open your application from the Profile tab of their Expo Go app.
+
+<ImageSpotlight alt="Terminal running expo init, with minimal (TypeScript) selected" src="/static/images/qr-code.gif" style={{height: '300px', width: 'auto'}} />
```

Comment:
video is an excellent addition 👍 but (as I found out just today) we should make this an mp4 (I have this file, can send it to you), and then use the Video component so that it matches the other screen recordings (like [this one](https://docs.expo.io/versions/latest/sdk/imagepicker/))